src/main/java/com/microsoft/graph/content/MSBatchRequestContent.java

```diff
@@ -0,0 +1,107 @@
+package com.microsoft.graph.content;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.http.Header;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpRequest;
+import org.apache.http.client.methods.HttpEntityEnclosingRequestBase;
+import org.apache.http.util.EntityUtils;
+import org.json.simple.JSONObject;
+import org.json.simple.JSONValue;
+
+public class MSBatchRequestContent {
+	private List<MSBatchRequestStep> batchRequestStepsArray;
+	private final int maxNumberOfRequests = 20;
+	
+	public MSBatchRequestContent(List<MSBatchRequestStep> batchRequestStepsArray) {
+		this.batchRequestStepsArray = new ArrayList<>();
+		if(batchRequestStepsArray.size() <= maxNumberOfRequests) {
+			for(MSBatchRequestStep requestStep: batchRequestStepsArray)
+				addBatchRequestStep(requestStep);
+		}
+	}
+	
+	public MSBatchRequestContent() {
+		batchRequestStepsArray = new ArrayList<>();
+	}
+	
+	public boolean addBatchRequestStep(MSBatchRequestStep batchRequestStep) {
+		if(batchRequestStep.getRequestId().compareTo("") == 0)
+			return false;
+		if(batchRequestStepsArray.size() == maxNumberOfRequests)
+			return false;
+		for(MSBatchRequestStep requestStep: batchRequestStepsArray) {
+			if(batchRequestStep.getRequestId().compareTo(requestStep.getRequestId()) == 0)
+				return false;
+		}
+		return batchRequestStepsArray.add(batchRequestStep);
+	}
+	
+	public void removeBatchRequesStepWithId(String requestId) {
```

Comment:
There is a possibility that the requestId passed to this function is not present in BatchRequestContent so similar to addBatchRequestStep function, this function could also have a boolean return type to indicate the success or failure of the operation.